src/main/java/org/apache/commons/lang3/ArrayUtils.java

```diff
@@ -802,8 +802,8 @@ private static Object add(final Object array, final int index, final Object elem
 
     /**
      * <p>Adds all the elements of the given arrays into a new array.
-     * <p>The new array contains all of the element of {@code array1} followed
-     * by all of the elements {@code array2}. When an array is returned, it is always
+     * <p>The new array contains all the element of {@code array1} followed
```

Comment:
This is still wrong: "The new array contains all the element of {@code array1} followed" -> "The new array contains all the element**s** of {@code array1} followed"

Reply:
I will make correct ，thanks